TypeCobol.LanguageServer/Completion Factory/CompletionFactory.cs

```diff
@@ -33,7 +34,8 @@ public static List<CompletionItem> GetCompletionPerformParagraph(FileCompiler fi
                 if (performNode.SymbolTable != null)
                 {
                     var userFilterText = userFilterToken == null ? string.Empty : userFilterToken.Text;
-                    pargraphs = performNode.SymbolTable.GetParagraphs(p => p.Name.StartsWith(userFilterText, StringComparison.InvariantCultureIgnoreCase));
+                    paragraphs = performNode.SymbolTable.GetParagraphs(p => p.Name.StartsWith(userFilterText, StringComparison.InvariantCultureIgnoreCase));
```

Comment:
Switch to OrdinalIgnoreCase for string comparison

Reply:
Done